### DISCUSSION TITLE
TTS Troubleshooting for Cast supported devices, when TTS is not heard.

### DISCUSSION THREAD

**Discussion by Jazzik777** [music-assistant/support#495] [Q&A]
1)    Confirm that TTS is working directly within the browser.  This can be done by selecting “Media” from the side bar. (If this is not present, hold down the sidebar button to display hidden options)

Ensure that your playback device (bottom right) is the browser. If you hear TTS from your PC this confirms that the TTS service (Google say or Nabu Casa) is working correctly.

2)    Check your internal/external URLs, as the TTS message is sent over your local LAN you must ensure that your internal IP address settings within home assistant are correct. This is even applicable if you are using Nabu Casa. (this is where I tripped up, I had migrated my HA from another internal device but hadn’t updated this setting) 

This can be found under:
Settings > System > Network > URL.

**Comment by erkr** [comment]
Another TTS issue can be the volume of the announcement. 

Good to know is that the TTS message is played with the default volume (0dB), where music is played with a reduced volume to allow for volume normalization. 
So the the LUFS setting (slider in the queue settings menu) directy impacts the relative TTS volume compared to the music. With -23 LUFS the TTS will be louder compared to the music than the default -14 LUFS.

**Comment by OzGav** [comment]
Relates to v1
